apps/web/src/pages/CommunityInterests/sections/AdditionalInformation.tsx

```diff
@@ -0,0 +1,92 @@
+import { useIntl } from "react-intl";
+import ClipboardDocumentCheckIcon from "@heroicons/react/24/outline/ClipboardDocumentCheckIcon";
+
+import { Heading } from "@gc-digital-talent/ui";
+import { RichTextInput } from "@gc-digital-talent/forms";
+import { getLocale } from "@gc-digital-talent/i18n";
+
+import { FRENCH_WORDS_PER_ENGLISH_WORD } from "~/constants/talentSearchConstants";
+
+const TEXT_AREA_MAX_WORDS_EN = 100;
+const TEXT_AREA_MAX_WORDS_FR = Math.round(
+  TEXT_AREA_MAX_WORDS_EN * FRENCH_WORDS_PER_ENGLISH_WORD,
+);
+
+export interface SubformValues {
+  additionalInformation: string | null;
+}
+
+interface AdditionalInformationProps {
+  formDisabled: boolean;
+}
+
+const AdditionalInformation = ({
+  // RichTextInput can't be disabled: #12531
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  formDisabled,
+}: AdditionalInformationProps) => {
+  const intl = useIntl();
+  const locale = getLocale(intl);
+
+  const wordLimits = {
+    en: TEXT_AREA_MAX_WORDS_EN,
+    fr: TEXT_AREA_MAX_WORDS_FR,
+  } as const;
+
+  return (
+    <div
+      data-h2-display="base(flex)"
+      data-h2-flex-direction="base(column)"
+      data-h2-gap="base(x1.25)"
+    >
+      {/* heading and description */}
+      <div
+        data-h2-display="base(flex)"
+        data-h2-flex-direction="base(column)"
+        data-h2-gap="base(x1)"
+      >
+        <Heading
+          level="h2"
+          data-h2-font-weight="base(400)"
+          Icon={ClipboardDocumentCheckIcon}
+          color="primary"
+          data-h2-margin="base(0)"
+        >
+          {intl.formatMessage({
+            defaultMessage: "Additional information",
+            id: "JFFBhZ",
+            description: "Heading for the 'Additional information' section",
+          })}
+        </Heading>
```

Comment:
Based on the content of the page, I'm leaning towards these being `level="h3"` but I'm not sure of the intent. This is the same for all headings in sub sections.

Reply:
I'm not sure how that would work?  The hero is h1.  If this is h3, what would h2 be?

Reply:
okay, that is where it was confusing for me. I assumed this was a sub section of "Find a new community" since we typically do that patter of always having a heading 2 under the heading 1 where everything on the page is a subsection of heading 2.

I think I understand now though